orchagent/portsorch.cpp

```diff
@@ -424,7 +424,7 @@ static bool isPathTracingSupported()
     attr.value.s32list.list = switchCapabilities.data();
 
     bool is_tam_supported = false;
-    auto status = sai_switch_api->get_switch_attribute(gSwitchId, 1, &attr);
+    sai_status_t status = sai_switch_api->get_switch_attribute(gSwitchId, 1, &attr);
```

Comment:
@DavidZagury why do we need to remove `auto` ?

Reply:
@nazariig 
The logs showed that the return value did in fact match SAI_STATUS_ATTR_NOT_IMPLEMENTED_0, but still has entered on some occasions to the error path and printed:

> ERR swss#orchagent: :- isPathTracingSupported: Failed to get a list of supported switch capabilities. Error=-196608

So that shouldn't have been printed, unless the comparison failed because of a mismatch that might be due to the variable types.